orkes-redis-queues/build.gradle

```diff
@@ -1,6 +1,7 @@
 dependencies {
 
     implementation "redis.clients:jedis:${versions.revJedis}"
+    implementation 'org.springframework.boot:spring-boot-starter'
```

Comment:
We would like to avoid bringing this in here.

Reply:
May I know why?

This dependency is required to initialise QueueMonitorProperties